src/components/CIPipelineN/TaskTypeDetailComponent.tsx

```diff
@@ -443,37 +440,6 @@ export function TaskTypeDetailComponent() {
         )
     }
 
-    const renderDockerScript = () => {
```

Comment:
is this intentional?

Reply:
Yes it was stale and was not using anywhere